src/common/components/clusterConfiguration/CertificateFields.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useField, useFormikContext } from 'formik';
+import { Grid } from '@patternfly/react-core';
+import { CheckboxField, HelperText, PopoverIcon, UploadField } from '../ui';
+import { TrustedCertificateFieldsType } from '../../types';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+
+import './ProxyFields.css';
+import { InfraEnv } from '../../api';
+
+export const CertificateFieldsHelperText: React.FC<{
+  fieldId?: string;
+}> = ({ fieldId = 'certificateField' }) => {
```

Comment:
I think it's best to export the constant `const FIELD_NAME = 'certificateField'` outside of the components and use it everywhere. Note that now `fieldId` is defined in this component, whereas in the main component its value `certificateField` is hardcoded.